patches/api/0009-Add-Position.patch

```diff
@@ -273,6 +274,13 @@ index 0000000000000000000000000000000000000000..300da713dcc303b340efad70efe57fac
 +    boolean isFine();
 +
 +    /**
++     * Checks if each component of this position is finite.
++     */
++    default boolean isFinite() {
++        return NumberConversions.isFinite(this.x()) && NumberConversions.isFinite(this.y()) && NumberConversions.isFinite(this.z());
```

Comment:
You can use the java equivalent with Double.isFinite

Reply:
Fixed.

Reply:
Fixed.